src/components/layout/PanelChange/index.tsx

```diff
@@ -83,7 +85,7 @@ export const PanelChange: FC<IProps> = ({
 	receivedSoFar,
 }) => {
 	const link = getAddressLink(contractAddress);
-	const { web3, address } = useShallowSelector(selectMain);
+	const { web3, address, linkHistory } = useShallowSelector(selectMain);
```

Comment:
Created a link history to keep track of what pages a user is visiting, this prevents aggregator randomly going off